pabot/result_merger.py

```diff
@@ -89,8 +90,7 @@ def visit_message(self, msg):
             if self._tests_root_name:
                 suites_names[0] = self._tests_root_name
             prefix = '.'.join(suites_names)
-            msg.message = msg.message.replace('selenium-screenshot',
-                                              prefix + '-selenium-screenshot')
+            msg.message = re.sub('"([^"]+.png)"', r'"%s-\1"' % prefix, msg.message)
```

Comment:
Same fix as above. "." equals anything so it must be escaped with "\".
Otherwise very good!